packages/docs/examples/Connectors.tsx

```diff
@@ -0,0 +1,123 @@
+/* eslint-disable no-console */
+import { Flex, Button, Input } from '@fuel-ui/react';
+import { useEffect, useState } from 'react';
+
+import type { FuelWalletConnector } from '~/../types/src';
+import { ExampleBox } from '~/src/components/ExampleBox';
+import { useFuel } from '~/src/hooks/useFuel';
+import { useLoading } from '~/src/hooks/useLoading';
+
+export function Connectors() {
+  const [fuel, notDetected] = useFuel();
+  const [connected, setConnected] = useState(false);
+  const [compatibilityError, setCompatibilityError] = useState('');
+  const [connectors, setConnectors] = useState<Array<FuelWalletConnector>>([]);
+
+  useEffect(() => {
+    if (!fuel) return () => {};
+    const onCurrentConnector = async () => {
+      console.log('Current connector changed!');
+      const isConnected = await fuel.isConnected();
+      setConnected(isConnected);
+    };
+    /* eventCurrentConnector:start */
+    fuel.on(fuel.events.currentConnector, onCurrentConnector);
+    return () => {
+      fuel.off(fuel.events.currentConnector, onCurrentConnector);
+    };
+    /* eventCurrentConnector:end */
+  }, [fuel]);
+
+  useEffect(() => {
+    if (!fuel) return () => {};
+    if (!fuel.listConnectors) {
+      setCompatibilityError(
+        'Current version of Fuel Wallet doesn't support Connectors!'
```

Comment:
```suggestion
        'Current version of Fuel Wallet doesn&apos;t support Connectors!'
```

Reply:
I fixed the string here. But not add &apos; as I like to avoid it on strings and keep it only when the text is directly on HTML.